config/initializers/swagger.rb

```diff
@@ -1,5 +1,5 @@
 unless Rails.env.production?
-  protocol = ENV["PORTUS_CHECK_SSL_USAGE_ENABLED"] ? "https://" : "http://"
+  protocol = APP_CONFIG.enabled?("check_ssl_usage") ? "https://" : "http://"
```

Comment:
Shouldn't it be `check_ssl_usage_enabled`?

Reply:
Just checked https://github.com/mssola/cconfig examples. But I'm not sure how `PORTUS_CHECK_SSL_USAGE_ENABLED` is parsed. Isn't it going to create something like `check -> ssl -> usage -> enabled -> value`?

Reply:
@vitoravelino I've added a test to `cconfig` just to show that it works :wink: Note that the "check_ssl_usage.enabled" value has already been merged with the value from the env. variable (`<prefix>_CHECK_SSL_ENABLED`) before the `enabled?` method is called (it's done with the railtie, thus on initialization).

Reply:
I didn't realize that there was a declared variable in the `config.yml` file. Now it makes sense. Thanks!